--- yubikey-val-2.39/Makefile.orig	2018-05-24 21:13:07.019601283 +0200
+++ yubikey-val-2.39/Makefile	2018-05-24 21:15:30.522932978 +0200
@@ -101,7 +101,7 @@
 	install -D ykval-munin-responses.pl $(DESTDIR)$(muninprefix)/ykval_responses
 	install -D ykval-munin-ksmresponses.pl $(DESTDIR)$(muninprefix)/ykval_ksmresponses
 	install -D ykval-munin-yubikeystats.php $(DESTDIR)$(muninprefix)/ykval_yubikeystats
-	install -D --backup --mode 640 --group $(wwwgroup) ykval-config.php $(DESTDIR)$(etcprefix)/ykval-config.php
+	install -D --mode 640 ykval-config.php $(DESTDIR)$(etcprefix)/ykval-config.php
 	install -D --mode 644 ykval-db.sql $(DESTDIR)$(docprefix)/ykval-db.sql
 	install -D --mode 644 ykval-db.oracle.sql $(DESTDIR)$(docprefix)/ykval-db.oracle.sql
 	install -D --mode 644 $(DOCS) $(DESTDIR)$(docprefix)/
